### ISSUE TITLE
Rest params in jsDoc

### ISSUE THREAD

**Issue by TxHawks** [eslint/eslint#5151]
I get a `valid-js` error When doing:

```js
/**
 * Function description
 * @param {...object} - Param description
 */
```

As per the [jsDoc documentation for @param](http://usejsdoc.org/tags-param.html#multiple-types-and-repeatable-parameters) (under `Allows a parameter to be repeated`), this should be valid.

**Comment by eslintbot** [comment]
@TxHawks Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

**Comment by TxHawks** [comment]
ESLint version: `1.10.3`

The relevant lines in my config file:
```js
{
  ....
  rules: {
    ...,
    'valid-jsdoc': [2, {
      requireReturn: false,
      requireParamDescription: false,
      requireReturnDescription: false
    }]
}
```

**Comment by gyandeeps** [comment]
Can you also share the exact error message you get?

**Comment by TxHawks** [comment]
Of course  -

```bash
124:3   error    JSDoc syntax error    valid-jsdoc
```

Line 124 is the beginning of the above comment block

**Comment by alberto** [comment]
@TxHawks can you provide a full example exposing the bug?

I tried the following on the [online demo](http://eslint.org/demo/) and I don't get any error.

```
/* eslint valid-jsdoc: [2, { "requireReturn": false }]  */

/**
 * Returns the sum of all numbers passed to the function.
 * @param {...number} num - A positive or negative number.
 */
function sum(num) {
}
```

**Comment by alberto** [comment]
Closing since I can't reproduce it. Please, open it again if you can provide more info.

**Comment by eslintbot** [comment]
Hi @TxHawks, thanks for the issue. It looks like there's not enough information for us to know how to help you. If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.

If it's something else, please just provide as much additional information as possible. Thanks!
